### DISCUSSION TITLE
Possible bug: Select street style in places requires become a sponsor

### DISCUSSION THREAD

**Discussion by MegaMaxIT** [photoprism/photoprism#2415] [Q&A] [answered]
Hi, I selected the offline style in places settings for trying it, but when I reselected the street style it ask me to become a sponsor for this option. It seems to me that isn't a sponsor feature, or not? When I installed photoprism the street style was activated, now I can't select it again.

It's a bug or really a sponsor's feature, what can I check?

The build is 220528-efb5d710 running in Openmediavault 6 x86 server

**Comment by lastzero** [comment, marked as answer]
Interactive World Maps are now a sponsor feature as we are paying for them ourselves and did not reach the initial funding goal even after more than 4 years. We are working on a complete feature comparison table, see Membership FAQ. It's also explicitly mentioned there that the maps are a sponsor feature:

https://photoprism.app/membership

**Reply by MegaMaxIT** [reply]
I had read that page but what was weird to me is that the same build when newly installed shows some kind of map and after it refuses to do so. I thought the interactive map was another feature. A new user may be confused, a table might be better :-)

Forgive me but I've only been using photoprism for a few days, thanks for the explanation.
